firefox-desktop/views/firefox_desktop_usage_2021.view.lkml

```diff
@@ -0,0 +1,166 @@
+view: firefox_desktop_usage_2021 {
+  sql_table_name: `mozdata.telemetry.firefox_desktop_usage_2021`
+    ;;
+
+  dimension: activity_segment {
+    type: string
+    sql: ${TABLE}.activity_segment ;;
+  }
+
+  dimension: campaign {
+    type: string
+    sql: ${TABLE}.campaign ;;
+  }
+
+  measure: cdou_computed {
+    type: running_total
+    sql: ${dau} ;;
+  }
+
+  measure: cdou {
+    type: sum
+    sql: ${TABLE}.cdou ;;
+  }
+
+  dimension: channel {
+    type: string
+    sql: ${TABLE}.channel ;;
+  }
+
+  dimension: content {
+    type: string
+    sql: ${TABLE}.content ;;
+  }
+
+  dimension: country {
+    type: string
+    map_layer_name: countries
+    sql: ${TABLE}.country ;;
+  }
+
+  measure: dau {
+    type: sum
+    sql: ${TABLE}.dau ;;
+  }
+
+  dimension: distribution_id {
+    type: string
+    sql: ${TABLE}.distribution_id ;;
+  }
+
+  dimension: id_bucket {
+    type: number
+    sql: ${TABLE}.id_bucket ;;
+  }
+
+  measure: mau {
+    type: sum
+    sql: ${TABLE}.mau ;;
+  }
+
+  dimension: medium {
+    type: string
+    sql: ${TABLE}.medium ;;
+  }
+
+  measure: new_profiles {
+    type: sum
+    sql: ${TABLE}.new_profiles ;;
+  }
+
+  measure: new_profiles_cumulative {
+    type: running_total
+    sql: ${new_profiles} ;;
+  }
+
+  dimension: os {
+    type: string
+    sql: ${TABLE}.os ;;
+  }
+
+  dimension: source {
+    type: string
+    sql: ${TABLE}.source ;;
+  }
+
+  dimension_group: submission {
```

Comment:
Does aggregating over non-submission-date time periods make sense for all these measures? For example, when I add the submission_month dimension and `mau` measure, I get the following query, which I believe would give a falsely inflated mau:
```
SELECT
	FORMAT_TIMESTAMP('%Y-%m', CAST(firefox_desktop_usage_2021.submission_date  AS TIMESTAMP)) AS firefox_desktop_usage_2021_submission_month,
	COALESCE(SUM(firefox_desktop_usage_2021.mau ), 0) AS firefox_desktop_usage_2021_mau
FROM `mozdata.telemetry.firefox_desktop_usage_2021`
     AS firefox_desktop_usage_2021

WHERE (CAST(CAST(firefox_desktop_usage_2021.submission_date  AS TIMESTAMP) AS DATE)) >= '2021-01-01'
GROUP BY 1
ORDER BY 1
LIMIT 500
```

Reply:
My rec would be to just include this as:
```
dimension: submission_date {
  type: date
  sql: ...
}
```

Reply:
(same for all other views)